tst/SignalingApiFunctionalityTest.cpp

```diff
@@ -257,6 +258,35 @@ TEST_F(SignalingApiFunctionalityTest, basicCreateConnectFree)
     EXPECT_EQ(STATUS_SUCCESS, freeSignalingClient(&signalingHandle));
 }
 
+TEST_F(SignalingApiFunctionalityTest, basicCreateCreateWithRetries)
```

Comment:
Nit: Naming seems weird. `basicCreate**Create**WithRetries` . Did you mean `basicCreateWithRetries`